retropie-shell-script-boilerplate.sh

```diff
@@ -1,33 +1,45 @@
 #!/usr/bin/env bash
 
-# [SCRIPT_TITLE] (e.g. Retropie Shell Script Boilerplate.)
+# [SCRIPT_NAME] (e.g.: script_template.sh)
 # [SCRIPT_DESCRIPTION] (e.g. A template for building shell scripts for RetroPie.)
 #
 # Author: [AUTHOR] (e.g. hiulit)
-# If you have the script in a GitHub Repository, use the comments below. If not, just remove them.
-# Repository: [REPOSITORY_URL] (e.g. https://github.com/hiulit/RetroPie-Shell-Script-Boilerplate)
-# Issues: [REPOSITORY_ISSUES_URL] (e.g. https://github.com/hiulit/RetroPie-Shell-Script-Boilerplate/issues)
+# Repository: [REPO_URL] (e.g. https://github.com/hiulit/RetroPie-Shell-Script-Boilerplate)
 # License: [LICENSE] [LICENSE_URL] (e.g. MIT https://github.com/hiulit/RetroPie-Shell-Script-Boilerplate/blob/master/LICENSE)
 #
 # Requirements:
 # - RetroPie x.x.x (e.g. RetroPie 4.x.x)
-# - [NAME_OF_THE_PACKAGE] package (e.g. libav-tools package)
+# - [PACKAGE_NAME] (e.g. libav-tools)
 
-home="$(find /home -type d -name RetroPie -print -quit 2>/dev/null)"
-home="${home%/RetroPie}"
+# globals ####################################################################
 
-readonly SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
-readonly SCRIPT_TITLE="[SCRIPT_TITLE]"
```

Comment:
Don't remove SCRIPT_TITLE and SCRIPT_DESCRIPTION, they are both used later on, in the help messages.

Reply:
OK

Reply:
I didn't want to sound too harsh, sorry if that was the case ;)